docs/plugins/filters/elastic_integration.asciidoc

```diff
@@ -0,0 +1,528 @@
+:plugin: elastic_integration
+:type: filter
+:default_plugin: 1
```

Comment:
@mashhurs, I know that you generated this locally for `main`.  This plugin will probably be bundled by default in `main`, but isn't yet.  Seems like maybe we should set the `default_plugin` value to `0` until then?  Please LMKWYT.

Reply:
Ah thanks for catching this up, it is due to my local and let me update it.

Reply:
Thanks for confirming. Nice to be sure that we're on the same page.